pkg/virtctl/usbredir/usbredir.go

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
If I understand correctly this command is valid only for linux because of `usbredirect`, right? You can add the build tag `+build linux`, so the command is not compiled for windows

Reply:
usbredirect can be used in windows too and there is a MSI instaler as well in the [release tag](https://gitlab.freedesktop.org/spice/usbredir/-/releases/usbredir-0.9.0). I have tested usbredirect on windows but I haven't tested virtctl usbredir on windows so it likely needs adjustments.

Reply:
sorry Victor, I mean the `usbredirectserver`. That is missing on windows right?

Reply:
`usbredirserver` binary is not used anymore, but indeed that one is linux only. Perhaps the const name `USBREDIRSERVER` is misleading in `pkg/virtctl/usbredir/usbredir.go`

Reply:
yes, thanks! I got confused by the name and the description of the PR  :)